README.md

```diff
@@ -1,6 +1,21 @@
 # kvproto
 Protocol buffer files for TiKV
 
+# Dependencies
+
+* Rust
+* Go
+* Protoc 3.1.0
+* This repo correctly placed in your `$GOPATH`.
+
+If you need to override your version of `protoc` because you have a later version you can do the following instead of `make` below:
+
+```bash
+PROTOC_VERSION=3.1.0
+curl -L https://github.com/google/protobuf/releases/download/v$PROTOC_VERSION/protoc-$PROTOC_VERSION-linux-x86_64.zip | unzip -p - bin/protoc > protoc-3.1.0
```

Comment:
It seems the `unzip` command doesn't work properly on my computer...? It outputs nothing.
By the way, I think the `include` directory in the zip file is also important

Reply:
Can you check for the file `protoc-3.1.0` to be created when you run that? It should not output anything on success.

I did not require the `include/` files.

Reply:
No, the file is empty, and `unzip` exits with exit code 9.

```
$ curl -L https://github.com/google/protobuf/releases/download/v$PROTOC_VERSION/protoc-$PROTOC_VERSION-linux-x86_64.zip > 1.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   154    0   154    0     0     98      0 --:--:--  0:00:01 --:--:--    98
100     9    0     9    0     0      4      0 --:--:--  0:00:01 --:--:--     4
$ ls
1.zip
$ unzip -p - bin/protoc < 1.zip
$ echo $?
9
$
```

Reply:
```
$ unzip -v
UnZip 6.00 of 20 April 2009, by Debian. Original by Info-ZIP.

Latest sources and executables are at ftp://ftp.info-zip.org/pub/infozip/ ;
see ftp://ftp.info-zip.org/pub/infozip/UnZip.html for other sites.

Compiled with gcc 5.2.1 20151119 for Unix (Linux ELF).
```

Reply:
Try setting the environment variable :)

Reply:
Oh, sorry, when I tried to post them I ran in to another mistake.

```
$ export PROTOC_VERSION=3.1.0
$ curl -L https://github.com/google/protobuf/releases/download/v$PROTOC_VERSION/protoc-$PROTOC_VERSION-linux-x86_64.zip > 1.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   164    0   164    0     0    120      0 --:--:--  0:00:01 --:--:--   120
100   619    0   619    0     0    358      0 --:--:--  0:00:01 --:--:--   358
100 1257k  100 1257k    0     0   226k      0  0:00:05  0:00:05 --:--:--  404k
$ unzip -p - bin/protoc < 1.zip
$ echo $?
9
$ ls
1.zip
$ unzip 1.zip
Archive:  1.zip
   creating: include/
   creating: include/google/
   creating: include/google/protobuf/
  inflating: include/google/protobuf/timestamp.proto
  inflating: include/google/protobuf/source_context.proto
  inflating: include/google/protobuf/any.proto
  inflating: include/google/protobuf/api.proto
  inflating: include/google/protobuf/descriptor.proto
  inflating: include/google/protobuf/struct.proto
  inflating: include/google/protobuf/wrappers.proto
   creating: include/google/protobuf/compiler/
  inflating: include/google/protobuf/compiler/plugin.proto
  inflating: include/google/protobuf/empty.proto
  inflating: include/google/protobuf/duration.proto
  inflating: include/google/protobuf/field_mask.proto
  inflating: include/google/protobuf/type.proto
   creating: bin/
  inflating: bin/protoc
  inflating: readme.txt
$ ls
1.zip  bin  include  readme.txt
$
```
It still exits with code 9. But if I run `unzip <filename>` directly, it can successfully unzip it.

Reply:
I clarified things a bit :)